subworkflows/local/lr_preprocessing.nf

```diff
@@ -0,0 +1,94 @@
+/*
+ * LR_PREPROCESSING: Preprocessing and QC for long reads
+ */
+
+include { NANOPLOT as NANOPLOT_RAW                              } from '../../modules/nf-core/nanoplot/main'
+include { NANOPLOT as NANOPLOT_FILTERED                         } from '../../modules/nf-core/nanoplot/main'
+include { NANOLYSE                                              } from '../../modules/nf-core/nanolyse/main'
+include { PORECHOP_PORECHOP                                     } from '../../modules/nf-core/porechop/porechop/main'
+include { PORECHOP_ABI                                          } from '../../modules/nf-core/porechop/abi/main'
+include { FILTLONG                                              } from '../../modules/nf-core/filtlong'
+
+workflow LR_PREPROCESSING {
```

Comment:
In the interests of verbosity should this be called LONG_READ_PREPROCESSING or similar?

Reply:
Agree, changed it to LONGREAD_PREPROCESSING